v1/ast/compile_test.go

```diff
@@ -736,7 +736,6 @@ func TestRuleTreeWithDotsInHeads(t *testing.T) {
 			c := NewCompiler()
 			for i, m := range tc.modules {
 				c.Modules[strconv.Itoa(i)] = m
-				c.sorted = append(c.sorted, strconv.Itoa(i))
```

Comment:
These weren't necessary: `compileStages` already does that. It's also harmful, since if c.sorted contains the same module twice, we'll run the stages concurrently on the same module, causing a data  race.